barretenberg/cpp/src/barretenberg/relations/relation_manual.test.cpp

```diff
@@ -4,16 +4,14 @@
 #include "barretenberg/relations/relation_parameters.hpp"
 #include <gtest/gtest.h>
 
-namespace bb::relation_manual_tests {
-
 using FF = bb::fr;
 
 class RelationManual : public testing::Test {};
 
 TEST_F(RelationManual, Poseidon2ExternalRelationZeros)
 {
     using Accumulator = std::array<FF, 4>;
-    using Relation = Poseidon2ExternalRelation<FF>;
+    using Relation = bb::Poseidon2ExternalRelation<FF>;
```

Comment:
I would think we would put even the tests in namespaces to avoid `using` or qualification boilerplate. Why not?

Reply:
I can go that route